include/radiopropa/IceModel.h

```diff
@@ -0,0 +1,110 @@
+#ifndef CRPROPA_ICEMODEL_H
+#define CRPROPA_ICEMODEL_H
+
+#include "radiopropa/Units.h"
+#include "radiopropa/Vector3.h"
+#include "radiopropa/ScalarField.h"
+
+#endif
+
+namespace radiopropa {
+
+class ExponentialIndex: public ScalarField
+{
+	protected:
+		double _n_ice, _delta_n, _z_shift, _z_0;
+	public:
+		ExponentialIndex(double n_ice, double delta_n, double z_0, double z_shift = 0);
+		virtual ~ExponentialIndex();
+		virtual double getValue(const Vector3d &position) const;
+		virtual double getAverageValue(const Vector3d &position1, const Vector3d &position2) const; 
+		virtual Vector3d getGradient(const Vector3d &position) const;
+};
+
+class IceModel_Simple: public ScalarField
+{
+	protected:
+		ExponentialIndex _ice;
+		double _z_surface;
+	public:
+		IceModel_Simple(double n_ice, double delta_n, double z_0, double z_shift = 0, double z_surface = 0);
+		virtual ~IceModel_Simple();
+		virtual double getValue(const Vector3d &position) const;
+		virtual double getAverageValue(const Vector3d &position1, const Vector3d &position2) const; 
+		virtual Vector3d getGradient(const Vector3d &position) const;
+};
+
+class IceModel_Firn: public ScalarField
+{
+	protected:
+		double _z_surface, _z_firn;
+		ExponentialIndex _firn;
+		ExponentialIndex _ice;
+	public:
+		IceModel_Firn(  
+			double n_ice_firn, 
```

Comment:
can you document here also the paper where the (firn)model is taken from? Like how is done for the gorham icemodel.

Reply:
why should this be documented? It is just to exponential stacked on each other but with no parameters, the parameters specific to the paper are implemented in NuRadioMC